test pytest and ruff

11909383 opened the pull request:
Modified two numbers in the test file and adjusted some spaces in the code to trigger a failure in Ruff."

11909383 committed:
Update test_playing_card_game.py

changed two numbers to fail the pytest

11909383 committed:
Update playing_card_game.py

changes some spaces to fail the ruff

11909383 commented:
@danielhuppmann

danielhuppmann commented:
Workflows approved. pytest failing as expected, but it seems that the extra empty lines are not sufficient to make ruff fail.

11909383 committed:
Update playing_card_game.py

I also added extra indentations and inserted unnecessary parentheses to fail ruff

11909383 commented:
I also added extra indentations and inserted unnecessary parentheses to fail ruff.

11909383 commented:
@danielhuppmann

danielhuppmann commented on playing_card_game.py:
```diff
@@ -1,28 +1,36 @@
 def deal_cards():
     # Create the deck of cards
-    deck = [f'{rank} of {suit}' for rank in ranks for suit in suits]
+        deck = [f'{rank} of {suit}' for rank in ranks for suit in suits]
```
This does not work as you expect. Indentation (at the start of a line) in python is very specific and serves to indicator start and end of functions, if-statements or loops. So with this change, ruff is still passing, but pytest is now failing with a "can't run this program because of illegal syntax" error.
